### ISSUE TITLE
Your OS reports the latest version of Plex (1.11.3.4803-c40bba82e) is already installed.

### ISSUE THREAD

**Issue by bdpna** [mrworf/plexupdate#224]
Plex server is reporting that version 1.12.0.4829 is available but the script is downloading 1.11.3.4803.

I have a plex pass.

Using openmediavault.

What am I doing wrong?

**Comment by bdpna** [comment]
Thanks so much for your quick response and diligence!

**Comment by derekcentrico** [comment]
Thanks for merging in.  Don't know how I missed this.

**Comment by demonbane** [comment]
No one from Plex has even acknowledged the post, but it does look like it's been fixed on their side, so if you were having this problem it should be solved **for now**.

I just created a pull request (#226) to at least prevent plexupdate from inadvertently downgrading you in case this happens again. (if you forced the update with `-p` but still had the cron job running nightly, you'd just get downgraded the same night)

I've also done some testing on a workaround just in case this ends up being a recurring problem. However, I'd prefer to not make substantial changes for something that could be a one-off, so we'll just wait and see if it becomes an issue again.

Closing this for now.

**Comment by bdpna** [comment]
You guys rule.

**Comment by derekcentrico** [comment]
I realize this is closed and, therefore, may not reopen or whatever.  I'm still in this quandery of no updates installing from PlexPass.  I know you mentioned not wanting to do much edits for a one-off but it isn't getting better for me at least.  (had a kid in the interim and just now noticing!).

> Retrieving list of available distributions
> Your OS reports the latest version of Plex (1.14.1.5488-cc260c476) is already installed. Use -f to force download.

However, the Plex webapp is telling me about a new fancy 1.15.xxxx version with many DVR fixes.

```
AUTOINSTALL='yes'
PUBLIC='no'
AUTOUPDATE='yes'
PLEXSERVER='127.0.0.1'
PLEXPORT='32400'
NOTIFY='yes'
AUTODELETE='yes'

```

**Comment by demonbane** [comment]
I did some digging around on the forums and found [this post](https://forums.plex.tv/t/plex-media-server/30447/245) announcing 1.15 with this bit at the bottom:

> See this post for links until the downloads page catches up

And a slightly longer bit [here](https://forums.plex.tv/t/where-is-version-1-15-0-647/372591/10):

> There were caching issues and the download pages were not showing the new version.

So the problem is literally that their downloads page just isn't updating like it should. The fixes I did before would help if the problem was that it was incorrectly tagged as a public release, but since it's just not there at all there's nothing we can do on our end.

If the update hasn't shown up by tomorrow I'd suggest commenting on one of those threads to see if someone on their side can kick the page update off manually or something, but hopefully it'll be resolved tonight.

**Comment by clovisd** [comment]
I've pinged it.

**Comment by demonbane** [comment]
Reopening this for now since it's still affecting multiple people. At this point we're just waiting on the Plex folks to figure out what is happening with their downloads page. This is the [forum thread](https://forums.plex.tv/t/linux-download-link-is-still-for-1-14/372535/39) that seems to have the most activity about this right now and also the one I'm keeping an eye on currently.

**Comment by demonbane** [comment]
Fix incoming for this. One of the Plex devs posted in the thread that they're switching to a new endpoint for 1.15 and later releases, so I've updated plexupdate to use that endpoint when checking for PlexPass releases. If you run into any problems with the new version please open a new issue.
